public/scripts/dropdown.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+// example for one button
+
+function handleMoreClick() {
+	let arr = ['classes', 'students'];
+
+	arr.forEach(function(item) {
+		$('.js-dropdown-btn-' + item).click(function(e) {
+			e.preventDefault();
+
+			$('.js-actions-dropdown-' + item).toggleClass('show');
+		});
+
+		window.onclick = function(e) {
```

Comment:
Desired functionality: user clicking anywhere in window other than on dropdown menu should be able to close that menu